frontend/site-registry/src/app/features/details/parcelDescriptions/getParcelDescriptions.ts

```diff
@@ -0,0 +1,96 @@
+import { print } from 'graphql';
+import gql from 'graphql-tag';
+import { GRAPHQL } from '../../../helpers/endpoints';
+import { getAxiosInstance } from '../../../helpers/utility';
+import {
+  IParcelDescriptionDto,
+  IParcelDescriptionResponseDto,
+} from './parcelDescriptionDto';
+import { format } from 'date-fns';
+
+export async function getParcelDescriptions(
+  siteId: number,
+  page: number,
+  pageSize: number,
+  searchParam: string,
+  sortBy: string,
+  sortByDir: string,
+) {
+  const axios = getAxiosInstance();
+  const response = await axios.post(GRAPHQL, {
+    query: print(gql`
+      query getParcelDescriptionBySiteId(
+        $siteId: Int!
+        $page: Int!
+        $pageSize: Int!
+        $searchParam: String!
+        $sortBy: String!
+        $sortByDir: String!
+      ) {
+        getParcelDescriptionsBySiteId(
+          siteId: $siteId
+          page: $page
+          pageSize: $pageSize
+          searchParam: $searchParam
+          sortBy: $sortBy
+          sortByDir: $sortByDir
+        ) {
+          message
+          httpStatusCode
+          success
+          count
+          page
+          pageSize
+          data {
+            id
+            descriptionType
+            idPinNumber
+            dateNoted
+            landDescription
+          }
+        }
+      }
+    `),
+    variables: {
+      siteId: siteId,
+      page: page,
+      pageSize: pageSize,
+      searchParam: searchParam,
+      sortBy: sortBy,
+      sortByDir: sortByDir,
+    },
+  });
+
+  let rawData = response?.data?.data?.getParcelDescriptionsBySiteId;
```

Comment:
We need to verify http response code before accessing content. Please refer to existing implementations such as folio or cart